core/web-assets/src/main/assets/js/apps/onms-daemons/index.html

```diff
@@ -0,0 +1,83 @@
+                <i class="fa fa-refresh fa-lg"></i> Refresh
+            </button>
+        </div>
+        <div ng-show="daemons.length == 0">No daemons available.</div>
+        <table ng-show="daemons.length > 0" class="table table-hover table-bordered table-striped table-condensed">
+            <thead>
+            <tr>
+                <th class="col-md-6 text-nowrap">Name</th>
+                <th class="col-md-1 text-nowrap text-center">Status</th>
+                <th class="col-md-1 text-nowrap text-center">Actions</th>
+                <th class="col-md-4 text-nowrap">Result</th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr ng-repeat="d in daemons">
+                <td>{{d.name}}</td>
+                <td class="text-center">
+                    <label ng-class="{'label label-success' : d.enabled, 'label label-danger' : !d.enabled}">
+                        <i ng-class="{'fa fa-pause' : !d.enabled, 'fa fa-play' : d.enabled}" title="{{d.enabled ? 'running' : 'not running'}}"/>
+                    </label>
+                </td>
+                <td class="text-center">
+                    <div ng-if="d.reloadable && d.enabled">
+                        <button ng-disabled="d.isReloading" type="button" ng-click="reloadDaemon(d)" class="btn btn-sm" title="Reload daemon configuration">
+                            <i class="fa fa-repeat fa-lg"/>
+                        </button>
+                        <button type="button" class="btn" data-toggle="modal" data-target="#curlModal_{{d.name}}" title="Show curl command to reload the configuration">
+                            <i class="fa fa-terminal fa-lg"></i>
+                        </button>
+                    </div>
+                </td>
+                <td>
+                    <span class="text-danger" ng-if="d.reloadState == 'Failed'"><i class="fa fa-times-circle"/></span>
+                    <span class="small" ng-if="d.reloadState == 'Failed'">An error occurred while processing the reload event</span>
+
+                    <span class="text-success" ng-if="d.reloadState == 'Success'"><i class="fa fa-check-circle"/></span>
+                    <span class="small" ng-if="d.reloadState == 'Success'">The daemon processed the reload event successfully</span>
+
+                    <span class="text-warning" ng-if="d.reloadState == 'Undefined'"><i class="fa fa-question-circle"/></span>
+                    <span class="small" ng-if="d.reloadState == 'Undefined'">It is unclear if the reload event was processed successfully</span>
+
+                    <span class="text-muted" ng-if="d.reloadState == 'Reloading'"><i class="fa fa-spinner fa-spin"/></span>
+                    <span class="small" ng-if="d.reloadState == 'Reloading'">Reloading...</span>
+                </td>
+
+                <!-- Modal -->
+                <div class="modal" id="curlModal_{{d.name}}" tabindex="-1" role="dialog" aria-labelledby="curlModalLabel">
```

Comment:
The modal should be implemented using one template only and a modalcontroller to set the variables accordingly